### PR TITLE
[web3] Fix Arguments to Match Web3 1.0 Docs

### PR DESCRIPTION
**Pull request by mikeseese**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://web3js.readthedocs.io/en/1.0/web3-eth.html#getstorageat and https://web3js.readthedocs.io/en/1.0/web3-utils.html#soliditysha3
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

Per the URLs above (and in the commits), this PR updates the `getStorageAt` and `soliditySha3` functions to reflect the 1.0 documentation.

### CODE REVIEW

**Comment by seesemichaelj** [comment]
The build failure is due to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28759, apparently there is a fix in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28794.

File: types/web3/web3-tests.ts [hunk 1]
```diff
@@ -22,3 +24,5 @@ const weiStr: string = web3.utils.toWei("100", "gwei");
 const weiBn: BigNumber = web3.utils.toWei(web3.utils.toBN("1"));
 const rndHex: string = Web3.utils.randomHex(10);
 const shhPromise: Promise<string> = web3.shh.generateSymKeyFromPassword("xyz");
+const storage: Promise<string> = web3.eth.getStorageAt("0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe", 0);
```
Comment (phra) [inline types/web3/web3-tests.ts]: you can reuse `contractAddress` variable instead of a hardcoded address.

**Review by phra — CHANGES_REQUESTED** [review]

File: types/web3/web3-tests.ts (hunk 1 above)
Comment (seesemichaelj) [reply types/web3/web3-tests.ts]: Woops, that was the intention haha. Thanks for catching that

Comment (seesemichaelj) [reply types/web3/web3-tests.ts]: I pushed the update, thanks again!

**Comment by seesemichaelj** [comment]
Closing and reopening to trigger a new travis build

**Review by phra — APPROVED** [review]

**Pull request merged**